### PR TITLE
FT2-868: Display reference code when saving progress.

### PR DESCRIPTION
**Pull request by ravi-methods**

### CODE REVIEW

File: DFC.App.SkillsHealthCheck.Services.SkillsCentral/Services/SkillsHealthCheckService.cs [hunk 1]
```diff
@@ -278,9 +280,9 @@ public DownloadDocumentResponse DownloadDocument(DownloadDocumentRequest downloa
                 response.DocumentBytes =
                     _skillsCentralService.FormatDocumentGetPayload(downloadDocumentRequest.DocumentId,
                         downloadDocumentRequest.Formatter);
-                    //ServiceHelper.Instance()
-                    // .Use<ISkillsCentralService, byte[]>(
-                    //     x => x.FormatDocumentGetPayload(downloadDocumentRequest.DocumentId, downloadDocumentRequest.Formatter));
+                //ServiceHelper.Instance()
```
Comment (AnatoliMalev-Methods) [inline DFC.App.SkillsHealthCheck.Services.SkillsCentral/Services/SkillsHealthCheckService.cs:283]: Just out of interest. How those commented out lines are going to work? And if are not intended to be used, they can be removed

**Review by AnatoliMalev-Methods — APPROVED** [review]

File: DFC.App.SkillsHealthCheck.Services.SkillsCentral/Services/SkillsHealthCheckService.cs (hunk 1 above)
Comment (ravi-methods) [reply DFC.App.SkillsHealthCheck.Services.SkillsCentral/Services/SkillsHealthCheckService.cs:283]: Not sure, not my code, I just formatted the document which changed alignment for it.

File: DFC.App.SkillsHealthCheck/Controllers/SaveMyProgressController.cs [hunk 2]
```diff
@@ -91,52 +102,154 @@ public IActionResult Body([FromQuery] string? type)
...
+            return this.NegotiateContentResult(new GetCodeViewModel
+            {
+                HtmlHeadViewModel = htmlHeadViewModel,
+                BreadcrumbViewModel = breadcrumbViewModel,
+                BodyViewModel = referenceViewModel,
+            });
+        }
+
+        [HttpPost]
+        [Route("skills-health-check/save-my-progress/getcode")]
```
Comment (andycyberian) [inline DFC.App.SkillsHealthCheck/Controllers/SaveMyProgressController.cs:160]: Have you tested this in shell, think you may need a 'skills-health-check/save-my-progress/getcode/body' route

**Review by andycyberian — COMMENTED** [review]
One comment with regard to routing in shell

File: DFC.App.SkillsHealthCheck/Controllers/SaveMyProgressController.cs (hunk 2 above)
Comment (ravi-methods) [reply DFC.App.SkillsHealthCheck/Controllers/SaveMyProgressController.cs:172]: Added another action method.

**Review by andycyberian — APPROVED** [review]

**Pull request merged**